renderer/src/actions/editingActions.ts

```diff
@@ -0,0 +1,19 @@
+export type EditingHistoryAction<T> =
+  | { type: "SET_VALUE"; value: T }
+  | { type: "UNDO" }
+  | { type: "REDO" };
+
+const setValue = (value: string): EditingHistoryAction<string> => ({
+  type: "SET_VALUE",
+  value,
+});
+
+const undo = (): EditingHistoryAction<string> => ({
```

Comment:
You have to put some meaning in the name of actions. Like this: undoEditing, redoEditing etc.

Reply:
Resolved